app/assets/javascripts/fae/form/_ajax.js

```diff
@@ -121,7 +121,21 @@ Fae.form.ajax = {
 
         // remotipart returns data inside textarea, let's grab it from there
         if ($html.is('textarea')) {
-          $html = $( $html.val() );
+          if ( $html.length > 1 ) {
+            var $textarea;
+
+            for (var i = 0; i < $html.length; i++ ) {
+              if ( $($html[i]).is('textarea') ) {
```

Comment:
Something seems wrong here. If, on L123, `$html === 'textarea'` then it should not be an Array. Does this code work in the dummy app?